_posts/2020-09-22-notes-from-community-call-3-september-2020.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Notes from community call - 3 September 2020"
+date: 2020-09-22
+author: Jan Ainali
+type: blogpost
+excerpt: Discussions about processes for how to do translations of the Standard for Public Code
+categories:
+  - Community call
+---
+
+# 3 September 2020 Standard for Public Code community call
+
+## Update from the Foundation
+
+* We finally sent out our first newsletter! You can also [sign up](https://forms.gle/gn7wR2Eaxbv5g1BF9).
+* 2 weeks ago, we presented on codebase stewardship at [Barcamp: Verwaltung. Digital. Gestalten](https://www.fokus.fraunhofer.de/de/dps/barcamp_200820), hosted by the Fraunhofer Institute for Open Communication Systems and NextNetz - this was our first presentation on codebase stewardship to a German audience.
+* Next week we will start our live stream / podcast [Let's talk about public code](https://blog.publiccode.net/news/2020/09/01/on-air-next-our-podcast.html).
+* Boris will present on why 'open source in gov't is different' at State of the Source on 9 September ([video](https://www.youtube.com/watch?v=mvhFF_7lVV4&list=PL_5ziu2gADmDcp_ER8x2rcT1HOBaIe5cl&index=3)).
+
+And for more news, you can always head to our [blog](https://blog.publiccode.net/).
+
+## Attendees
+
+* [Alba Roza](https://publiccode.net/team/alba-roza.html) (Chair)
+* [Jan Ainali](https://publiccode.net/team/jan-ainali.html)
+* [Eric Herman](https://publiccode.net/team/eric-herman.html)
+* [Boris van Hoytema](https://publiccode.net/team/boris-van-hoytema.html)
+* Marc van Andel
+* Martin Dias d'Ullios
+* Matti Schneider
+* Olaf-Gerd Gemein
+
+## Notes
+
+National public organizations might have access to professional translators in the EU languages which opens up a totally different path than traditional community translations by volunteers.
+The Foundation for Public Code could then be the one that take institutional responsibility for overseeing and merging translations in collaboration with a community for each language.
+This solution seemed particulary interesting and the Foundation for Public Code will look into this path next.
+
+As a short term solution the Standard for Public Code could also be modularized further.
+Of particular interest would be an executive summary that also would be quicker to translate and at the same time might be the most important part to localize.
+
+During the meeting a summary of the previous call and how the Foundation for Pulic Code have been thinking about translations was given.
+Jan described how it will be a lot easier for people just doing translations if platforms purposed for this are used (having translation memory, documentation about what's being translated etc.) and how those can be automated to commit to a git repository.
```

Comment:
```suggestion
Jan described how it would be a lot easier for people just doing translations if platforms designed for this are used (with features like translation memory or documentation about what's being translated) and how those can be automated to commit to a git repository.
```